src/app/direct-debit/page.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author David Zhu <dzhu292@aucklanduni.ac.nz>
+ */
+
+"use client";
+
+import Heading from "@/components/Heading/Heading";
+import PaymentInfoCard from "@/components/PaymentInfoCard/PaymentInfoCard";
+import DebitDetailsCard from "@/components/DebitDetailsCard/DebitDetailsCard";
+import Button from "@/components/Button/Button";
+import ScrollShadow from "@/components/ScrollShadow";
+
+export default function DirectDebitPage() {
+  const firstName: string = "John";
+  const lastName: string = "Smith";
+  const sessionId: string = "rn3498";
+  const accountNumber: string = "xx-xxxx-xxxx-xxx";
```

Comment:
Just note when you have string defined like this typescript can infer the type. Not a deal breaker but you don't always have to strictly type like this